metricbeat/mb/testing/data_generator.go

```diff
@@ -62,7 +62,7 @@ func WriteEvents(f mb.EventsFetcher, t testing.TB) error {
 // This simulates the output of Metricbeat as if it were
 // 2016-05-23T08:05:34.853Z and the hostname is host.example.com.
 func CreateFullEvent(ms mb.MetricSet, metricSetData common.MapStr) beat.Event {
-	startTime, err := time.Parse(time.RFC3339Nano, "2016-05-23T08:05:34.853Z")
+	startTime, err := time.Parse(time.RFC3339Nano, "2017-09-25T08:05:34.853Z")
```

Comment:
Let me revert this part of the change, so we can investigate in a follow up PR on why this change does not apply and how to cleanup / improve the generation.